laskyawm.css

```diff
@@ -145,6 +138,19 @@ body {
 .window.maximized.maximized-max, .hud-max {
 	left: 100px !important;
 	width: calc(100% - 100px) !important;
+	border-radius: 0;
+}
+.window.maximized.maximized-max .titlebar {
+	position: fixed;
+	top: 0;
+	left: 100px;
+	width: 100%;
+}
+.window.maximized.maximized-max .tabs {
+	height: 100%;
+}
+.window.maximized.maximized-max iframe {
+	border-radius: 0;
```

Comment:
I agree the maximized window thing is cool, but it's on top of the minimized windows now. Maybe hold off on this for now?